sqlconnect/op/operator.go

```diff
@@ -0,0 +1,47 @@
+package op
+
+type Operator string
+
+const (
+	// =
+	Eq Operator = "eq"
+	// !=
+	Neq Operator = "neq"
+	// IN (...)
+	In Operator = "in"
+	// NOT IN (...)
+	NotIn Operator = "notin"
+
+	// >
+	Gt Operator = "gt"
+	// >=
+	Gte Operator = "gte"
+	// <
+	Lt Operator = "lt"
+	// <=
+	Lte Operator = "lte"
+
+	// LIKE
+	Like Operator = "like"
+	// NOT LIKE
+	NotLike Operator = "notlike"
+	// left <= v <= right
+	Between Operator = "between"
```

Comment:
For Between, Should we think of some way to specify left and right bounded/unbounded conditions.

Reply:
Also NotBetween

Reply:
What about
- btw, nbtw --> between & not between
- btwe, nbtwe --> between or equal & not between or equal